### PR TITLE
Azure devops doc, restructure

### PR DESCRIPTION
**Pull request by mclacore**

### CODE REVIEW

File: docs/ci_cd/01-github.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ This guide will walk you through the process of setting up a GitHub Action to bu
 Before getting started, you'll need:
 - A Massdriver account 
 - A Massdriver [service account](/platform/service-accounts)
+- A GitHub repository with your application code
```
Comment (coryodaniel) [inline docs/ci_cd/01-github.md:17]: can you toss a link to register on line 15?

File: docs/ci_cd/02-devops.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+---
+id: ci-cd-devops
```
Comment (coryodaniel) [inline docs/ci_cd/02-devops.md:2]: we may want to call this azure-devops so it doesn't collide within anything

File: docs/ci_cd/01-github.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-id: applications-container-repositories-github
-slug: /applications/container-repositories/github
+id: ci-cd-github
```
Comment (coryodaniel) [inline docs/ci_cd/01-github.md:2]: may want to call this -github-actions so its inline w/ azure note below

**Review by coryodaniel — COMMENTED** [review]
Two notes. Also there is another "CI/CD" page https://docs.massdriver.cloud/bundles/github-action ... are we leaving that there? 

seems kinda right as its the next step in bundle publish ... but should we add azure docs there too?

File: docs/ci_cd/01-github.md (hunk 1 above)
Comment (mclacore) [reply docs/ci_cd/01-github.md:17]: Register what? Their github repo?

To install our GitHub action, it just gives you the step needed to install:
image

**Comment by mclacore** [comment]
> Two notes. Also there is another "CI/CD" page https://docs.massdriver.cloud/bundles/github-action ... are we leaving that there?
> 
> seems kinda right as its the next step in bundle publish ... but should we add azure docs there too?

Oh right I was merging both app and infra into both, I'll add that in.

File: docs/ci_cd/01-github.md (hunk 1 above)
Comment (coryodaniel) [reply docs/ci_cd/01-github.md:17]: Massdriver registration page!

**Review by coryodaniel — APPROVED** [review]
Didnt test examples, but docs look g2g

**Pull request merged**